build.py

```diff
@@ -50,9 +50,10 @@
     description='Command Line Interface for managing PROS projects',
     options={"build_exe": build_exe_options, 'bdist_mac': build_mac_options},
     install_requires=install_reqs,
-    executables=[Executable('pros/cli/main.py', targetName=f'prosv5{extension}'),
-                 Executable('pros/cli/compile_commands/intercept-cc.py', targetName=f'intercept-cc{extension}'),
-                 Executable('pros/cli/compile_commands/intercept-cc.py', targetName=f'intercept-c++{extension}')]
+    executables=[Executable('pros/cli/main.py', target_name=f'pros{extension}'),
+                 Executable('pros/cli/main.py', target_name=f'prosv5{extension}'),
+                 Executable('pros/cli/compile_commands/intercept-cc.py', target_name=f'intercept-cc{extension}'),
+                 Executable('pros/cli/compile_commands/intercept-cc.py', target_name=f'intercept-c++{extension}')]
```

Comment:
changed targetName to target_name here because of this cx_Freeze deprecation notice
```
/home/hotel/.pyenv/versions/pros-cli-build/lib/python3.7/site-packages/cx_Freeze/common.py:125: UserWarning: camelCase values is obsolete and will be removed in the next major version -> use the new name 'target_name'
  "camelCase values is obsolete and will be removed in the "
```